05LessonFive/Checkpoint1/index.html

```diff
@@ -1,10 +1,24 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Landscape Design</title>
+  <title>Berries</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div class="berryimg">
+    <img src="./img/berries.jpg">
+  </div>
+
+  <div class="berryparagraph">
+    <span><h2>Berries</h2>
```

Comment:
I tried, but it moved the heading and text to the top of the div instead of keeping it in the middle. I was honestly stuck on how to get it to the center of the div for the longest time; the <span> ended up getting me the results I wanted, haha.